kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/api/common/Condition.java

```diff
@@ -6,6 +6,12 @@
 
 package io.kroxylicious.kubernetes.api.common;
 
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.Nulls;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
We said we've favour package-info.java defaults for establishing NonNull.

Reply:
`edu.umd.cs.findbugs.annotations.NonNull` doesn't support package level. So we need to switch our annotation of choice to do so.  

https://github.com/kroxylicious/kroxylicious/pull/2017#discussion_r2025841891

Reply:
the config we've got elsewhere is, for example: 

```
@ReturnValuesAreNonnullByDefault
@DefaultAnnotationForParameters(NonNull.class)
@DefaultAnnotation(NonNull.class)
package io.kroxylicious.kubernetes.operator.resolver;

import edu.umd.cs.findbugs.annotations.DefaultAnnotation;
import edu.umd.cs.findbugs.annotations.DefaultAnnotationForParameters;
import edu.umd.cs.findbugs.annotations.NonNull;
import edu.umd.cs.findbugs.annotations.ReturnValuesAreNonnullByDefault;
```